ocaml/xapi/message_forwarding.ml

```diff
@@ -2328,6 +2302,14 @@ functor
                 )
             )
         ) ;
+        let message_body =
+          Printf.sprintf "VM '%s' migrated from host '%s' to host '%s'"
+            (Db.VM.get_name_label ~__context ~self:vm)
+            (Db.Host.get_name_label ~__context ~self:source_host)
```

Comment:
Should this report the UUIDs, too?

Reply:
I can add it, the uuid was not in pre-existing messages, but IMHO adding it is a good idea as we have no insurance name_label is unique per vm. Should i do it?

Reply:
Yeah, go for it, I'm sure @stormi will like that

Reply:
Done :+1: I've added the uuids of the hosts in the migration messages as well :)